docker-entry.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # migration
-prisma migrate dev
+prisma generate
+npx prisma migrate dev
```

Comment:
should use `prisma migrate deploy` command for running migration in production.

check [here](https://www.prisma.io/docs/concepts/components/prisma-migrate#production-and-testing-environments) for more info

@Guruprasanna02